<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomEditor</class>
 <widget class="QMainWindow" name="RoomEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit room</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QDockWidget" name="dckSettings">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>225</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Toolbox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QScrollArea {
border: 0px;
margin: 0px;
}</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>182</width>
          <height>445</height>
         </rect>
        </property>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>290</y>
           <width>181</width>
           <height>101</height>
          </rect>
         </property>
         <property name="title">
          <string>Room redraw</string>
         </property>
         <widget class="QPushButton" name="btnVoidColor">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>91</width>
            <height>30</height>
           </rect>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>51</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chkFillVoid">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>161</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Fill void with color</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Room name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtRoomName">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>181</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QTabWidget" name="tbwTabs">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>160</y>
           <width>201</width>
           <height>121</height>
          </rect>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabObjects">
          <attribute name="title">
           <string>Objects</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>181</width>
             <height>71</height>
            </rect>
           </property>
           <property name="title">
            <string>Place object</string>
           </property>
           <widget class="QComboBox" name="cmbObjectSel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tabBG">
          <attribute name="title">
           <string>Tiles</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>181</width>
             <height>71</height>
            </rect>
           </property>
           <property name="title">
            <string>Place background tiles</string>
           </property>
           <widget class="QComboBox" name="cmbTIleSel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>181</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Room size</string>
         </property>
         <widget class="QSpinBox" name="spbRoomHeight">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>50</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spbRoomWidth">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>49</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dckTop">
   <property name="minimumSize">
    <size>
     <width>29</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>40</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/ok</normaloff>:/icons/ok</iconset>
     </property>
    </widget>
    <widget class="QSpinBox" name="spbSnapWidth">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Snap:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spbSnapHeight">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkGrid">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw grid</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>0</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
